elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -0,0 +1,144 @@
+elytron-oidc-client.secure-deployment.token-minimum-time-to-live=The adapter will refresh the token if the current token is expired OR will expire in 'token-minimum-time-to-live' seconds or less
+elytron-oidc-client.secure-deployment.min-time-between-jwks-requests=If adapter recognize token signed by unknown public key, it will try to download new public key from elytron-oidc-client server. However it won't try to download if already tried it in less than 'min-time-between-jwks-requests' seconds
+elytron-oidc-client.secure-deployment.public-key-cache-ttl=Maximum time the downloaded public keys are considered valid. When this time reach, the adapter is forced to download public keys from elytron-oidc-client server
+elytron-oidc-client.secure-deployment.ignore-oauth-query-parameter=disable query parameter parsing for access_token
+elytron-oidc-client.secure-deployment.proxy-url=The URL for the HTTP proxy if one is used.
+elytron-oidc-client.secure-deployment.verify-token-audience=If true, then during bearer-only authentication, the adapter will verify if token contains this client name (resource) as an audience
+elytron-oidc-client.secure-deployment.adapter-state-cookie-path=If set, defines the path used in cookies set by the adapter. Useful when deploying the application in the root context path.
+elytron-oidc-client.secure-deployment.credential=Credential used to communicate with the OpenID Connect provider
+elytron-oidc-client.secure-deployment.credential.add=Add credentials used to communicate with the OpenID Connect provider
+elytron-oidc-client.secure-deployment.credential.remove=Remove credentials used to communicate with the OpenID Connect provider
+elytron-oidc-client.secure-deployment.credential.client-key-password=Password for the client key. This is required if client-keystore-file has been specified.
+elytron-oidc-client.secure-deployment.credential.client-keystore-password=Password for the client keystore. This is required if client-keystore-file has been specified.
+elytron-oidc-client.secure-deployment.credential.client-keystore-file=Path to the client keystore. This is optional.
+elytron-oidc-client.secure-deployment.credential.client-keystore-type=Client keystore type
+elytron-oidc-client.secure-deployment.credential.client-key-alias=Client key alias
+elytron-oidc-client.secure-deployment.credential.token-timeout=Token timeout
+elytron-oidc-client.secure-deployment.credential.secret=Secret
+elytron-oidc-client.secure-deployment.credential.algorithm=Algorithm for signed JWT
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.add=Add a rewrite rule for the redirect URI
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.remove=Remove a rewrite rule for the redirect URI
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.replacement=The replacement string
+elytron-oidc-client.secure-deployment.socket-timeout-millis=Timeout for socket waiting for data in milliseconds
```

Comment:
@fjuma Just a minor but these last 3 are duplicates

Reply:
@Skyllarr I've removed the duplicates, thanks!